tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/repopulate_person_controlled_tier_test.py

```diff
@@ -106,20 +107,25 @@ def setUp(self):
                 value_as_concept_id,
                 observation_source_concept_id,
                 value_source_concept_id,
+                value_source_value,
                 observation_date,
                 observation_concept_id,
                 observation_type_concept_id
             )
             VALUES
-                (1, 1, 45877987, 1586140, 1586146, '2020-01-01', 0, 0),
-                (2, 1, 1586143, 1586140, 1586143, '2020-01-01', 0, 0),
-                (3, 2, 45879439, 1586140, 1586142, '2020-01-01', 0, 0),
-                (4, 2, 1586147, 1586140, 1586147, '2020-01-01', 0, 0),
-                (5, 1, 45878463, 1585838, 1585840, '2020-01-01', 0, 0),
-                (6, 2, 45880669, 1585838, 1585839, '2020-01-01', 0, 0),
-                (7, 2, 1585841, 1585838, 1585841, '2020-01-01', 0, 0),
-                (8, 1, 45878463, 1585845, 1585847, '2020-01-01', 0, 0),
-                (9, 2, 45880669, 1585845, 1585846, '2020-01-01', 0, 0)
+                (1, 1, 45877987, 1586140, 1586146, 'WhatRaceEthnicity_White', '2020-01-01', 0, 0),
+                (2, 1, 1586143, 1586140, 1586143, 'WhatRaceEthnicity_Black', '2020-01-01', 0, 0),
+                (3, 2, 45879439, 1586140, 1586142, 'WhatRaceEthnicity_Asian', '2020-01-01', 0, 0),
+                (4, 2, 1586147, 1586140, 1586147, 'WhatRaceEthnicity_Hispanic', '2020-01-01', 0, 0),
+                (5, 1, 45878463, 1585838, 1585840, 'GenderIdentity_Woman', '2020-01-01', 0, 0),
+                (6, 2, 45880669, 1585838, 1585839, 'GenderIdentity_Man', '2020-01-01', 0, 0),
```

Comment:
Tests are great! Thanks for adding the `value_source_value`, makes it easier to understand.